api/integrations/github/constants.py

```diff
@@ -0,0 +1,3 @@
+GITHUB_API_URL = "https://api.github.com/"
+GITHUB_API_VERSION = "2022-11-28"
+GITHUB_APP_ID = 811209
```

Comment:
Is this static? Or should we move this to settings?

Reply:
These variables won't change; they can be constants

Reply:
Think the GITHUB_APP_ID will change if it's used in private cloud?

Reply:
Yes, I will move this out to external config and read it in the settings like the rest of external configurations

Reply:
GITHUB_APP_ID moved to settings